lib/evm/eei.ts

```diff
@@ -369,7 +369,7 @@ export default class EEI {
       value: value,
       data: data,
       isStatic: this._env.isStatic,
-      depth: this._runState.depth + 1
+      depth: this._env.depth + 1
```

Comment:
Not getting this ``depth`` to ``env`` change - at least not directly intuitively - since I would place this on first thought to the run state, could you give a short explanation?

Reply:
I moved it to `env` to stress the fact that it should not be modified in a single call context. It is incremented each time a new call (or create) is invoked.